ckan/Dockerfile

```diff
@@ -58,7 +58,7 @@ USER ckan
 ARG CKAN_BRANCH
 ARG CKAN_REPO
 
-RUN CKAN_BRANCH="${CKAN_BRANCH:-ckan-2.8.1}" && CKAN_REPO="${CKAN_REPO:-ckan/ckan}" &&\
+RUN CKAN_BRANCH="${CKAN_BRANCH:-ckan-2.7.3}" && CKAN_REPO="${CKAN_REPO:-ckan/ckan}" &&\
```

Comment:
Let's have 2.8.1 (at least here). It will be overridden if ser in .docker-compos.philiphins..